src/impls/slab.rs

```diff
@@ -14,6 +14,8 @@ use crate::{
 
 impl<T> Collection for Slab<T> {
 	type Item = T;
+	type ItemRef<'a> where Self: 'a = &'a T;
```

Comment:
This is not compatible with the new definition of `Collection`. I should add a github workflow to spot this kind of issues automatically...